### ISSUE TITLE
Cannot read property 'set' of undefined error

### ISSUE THREAD

**Issue by yuvalsabar** [avocode/avocode#1848]
---
* **Version**: 2.18.2
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 232748
---
```
TypeError: Cannot read property 'set' of undefined
    at t.value (file:///C:/Users/yuval/AppData/Local/avocode/app-2.18.2/resources/app.asar/dist/avocode.min.js:1:1678220)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (C:\Users\yuval\AppData\Local\avocode\app-2.18.2\resources\app.asar\node_modules\react-dom\lib\ReactCompositeComponent.js:799:30)
    at ReactCompositeComponentWrapper._renderValidatedComponent (C:\Users\yuval\AppData\Local\avocode\app-2.18.2\resources\app.asar\node_modules\react-dom\lib\ReactCompositeComponent.js:822:32)
    at ReactCompositeComponentWrapper.performInitialMount (C:\Users\yuval\AppData\Local\avocode\app-2.18.2\resources\app.asar\node_modules\react-dom\lib\ReactCompositeComponent.js:362:30)
    at ReactCompositeComponentWrapper.mountComponent (C:\Users\yuval\AppData\Local\avocode\app-2.18.2\resources\app.asar\node_modules\react-dom\lib\ReactCompositeComponent.js:258:21)
    at Object.mountComponent (C:\Users\yuval\Ap
```

**Comment by vojtatranta** [comment]
Thank you for this report, this looks like genuine error. We will take a look at it in the next release.

**Comment by vojtatranta** [comment]
This bug appears to be fixed in `2.19.x`
